docs/user-guide.md

```diff
@@ -0,0 +1,76 @@
+# User Guide
+
+## Workflows
+
+## Verify Kani harnesses
```

Comment:
Is `Verify Kani harnesses` a sub-heading of `Workflows`? If yes, please use different heading level.